doc/requirements/io_class

```diff
@@ -0,0 +1,86 @@
+---
+group: IO class
+---
+
+Requirements for IO classes
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting occupancy limit
+id: occupancy
+---
+
+It shall be possible to set a maximum size of an IO class.
```

Comment:
I not sure, but I think it should be 'the maximum size of an IO class' - 'an IO class' is ok, because it's one of many, but an IO class has only one 'maximum size'. Am I overzealous in this?

Reply:
English articles are dark and full of terrors 😅

Reply:
"the maximum size of an IO class" sounds definitely wrong to me. there are multiple possible maximum values, not one.